std/http/server.ts

```diff
@@ -211,8 +211,12 @@ export class Server implements AsyncIterable<ServerRequest> {
     try {
       conn = await this.listener.accept();
     } catch (error) {
-      if (error instanceof Deno.errors.BadResource) {
-        return;
+      if (
+        error instanceof Deno.errors.BadResource ||
```

Comment:
Adding `ConnectionAborted` might also fix https://github.com/denoland/deno/issues/6013

We should probably add all possible errors for `this.listener.accept()` that should not close the server. Or maybe there isn't an error that should close it, and just: 

```javascript
try {
    conn = await this.listener.accept();
} catch(error) {
    // Accept another connection
    return mux.add(this.acceptConnAndIterateHttpRequests(mux));
}
```

Reply:
@marcosc90 what was the justification for scoring out your comment r.e. `ConnectionAborted`? Ask because folks are suggesting that this get's handled in wrapping frameworks ( REF: https://github.com/asos-craigmorten/opine/pull/124 ) and curious what led to not adding it in?